SmartDeviceLinkTests/DevAPISpecs/SDLLockScreenManagerSpec.m

```diff
@@ -233,7 +233,31 @@ @interface SDLLockScreenManager ()
             });
         });
     });
-    
+
+    context(@"with showDeviceLogo as false",  ^{
+        beforeEach(^{
+            SDLLockScreenConfiguration *config = [SDLLockScreenConfiguration enabledConfiguration];
+            config.showDeviceLogo = NO;
+
+            testManager = [[SDLLockScreenManager alloc] initWithConfiguration:config notificationDispatcher:nil presenter:fakePresenter];
+            [testManager start];
+        });
+
+        describe(@"when a vehicle icon is received", ^{
+            __block UIImage *testIcon = nil;
+
+            beforeEach(^{
+                testIcon = [UIImage imageNamed:@"testImagePNG" inBundle:[NSBundle bundleForClass:self.class] compatibleWithTraitCollection:nil];
+                [[NSNotificationCenter defaultCenter] postNotificationName:SDLDidReceiveLockScreenIcon object:nil userInfo:@{ SDLNotificationUserInfoObject: testIcon }];
+            });
+
+            it(@"should have a vehicle icon if showDeviceLogo is set to true", ^{
```

Comment:
It should be true, but the test says it's nil. I don't think this test is working.

Reply:
Im sorry I completely miss typed the `IT` and `Context` this is testing that when set to NO the icon should not be set. I fixed the description. the reverse of this test is tested on line 136 since 'showDeviceLogo` defaults to YES